docs/deployments/debugging-harperdb.md

```diff
@@ -0,0 +1,44 @@
+# Debugging HarperDB
```

Comment:
I think somewhere up here we need to add some context around what kinds of debugging this is referring to. @kriszyp This information would be specifically for self-managed deployments, correct?

Reply:
Yes, this is for remote connection to servers that you have SSH access to and access to the ports.